### PR TITLE
Geiger Counter Sound Fix

### PR DESCRIPTION
**Pull request by ThatDan123**
Makes Sound local to geiger Counter, not global.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tool/GeigerCounter.cs [hunk 1]
```diff
@@ -57,19 +58,19 @@ public void CycleUpdate()
 		if (node  == null) return;
 		if (node.RadiationNode.RadiationLevel > 1000)
 		{
-			SoundManager.PlayNetworked(Noise[Level.Extreme][RNG.Next(0,3)]);
+			SoundManager.PlayNetworkedAtPos(Noise[Level.Extreme][RNG.Next(0,3)], registerItem.WorldPositionServer, sourceObj: gameObject);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Tool/GeigerCounter.cs:61]: forgot to say if this is in someone's inventory will return hidden POS

Comment (vlandemart) [inline UnityProject/Assets/Scripts/Tool/GeigerCounter.cs:61]: "forgot to say if this is in someone's inventory will return hidden POS" - Needs to be fixed

**Review by vlandemart — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Scripts/Tool/GeigerCounter.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tool/GeigerCounter.cs:61]: Yeah sorry, was just finishing up the lava land stuff to a reasonable place before coming back to fix this.

**Comment by ThatDan123** [comment]
Just did a two client test and this seems to work even if its in a pocket

**Comment by Bod9001** [comment]
interesting,  I will merge

**Pull request merged**